releases/release-1.31/release-notes/maps/pr-126237-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 126237
+releasenote:
+  text: Promoted the metrics for both VAP and CRD validation rules to Beta.
```

Comment:
```suggestion
  text: Promoted the metrics for both Validating Admission Policy (VAP) and Custom Resource Definition (CRD) validation rules to beta.
```

Reply:
I modified "beta" to "Beta" everywhere else. Are we going to update all to all lowercase?

Reply:
I think we decided on lowercase last time? https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653189507